adapter/src/dbt/adapters/fal_experimental/adapter.py

```diff
@@ -80,7 +81,11 @@ def run_in_environment_with_adapter(
             raise RuntimeError(
                 "BigQuery credential method `service-account` is not supported." + \
                 " Please use `service-account-json` instead")
-        deps = [i for i in get_default_pip_dependencies(is_remote=True) if i.startswith('dbt-')]
+        deps = [
+            i
+            for i in get_default_pip_dependencies(is_remote=True, adapter_type=adapter_type)
+            if i.startswith('dbt-')
```

Comment:
atl: why is it that we have to filter `dbt-`

Reply:
Updated the code to handle this in environments.py